rpcserver.go

```diff
@@ -2798,12 +2799,10 @@ func createRPCOpenChannel(r *rpcServer, graph *channeldb.ChannelGraph,
 		lifespan = endTime.Sub(startTime)
 
 		uptime, err = r.server.chanEventStore.GetUptime(
-			chanID,
-			startTime,
-			endTime,
+			dbChannel.FundingOutpoint, startTime, endTime,
 		)
 		if err != nil {
-			rpcsLog.Warnf("GetUptime %v error: %v", chanID, err)
+			rpcsLog.Warnf("GetUptime %v error: %v", dbChannel.FundingOutpoint, err)
```

Comment:
Is every error only a warning? Or only the channel not found error?

Reply:
This can probably be upgraded to an error, if we get channel not found then there's a bug somewhere because all the channels returned by `listchannels` should be tracked in chafitness. 

Likewise, the other errors would be from incorrect the start/end times, which is also a bug.